authors/settings.py

```diff
@@ -24,8 +24,6 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
```

Comment:
@Emmanuelaaron  l would like to know why did you remove line 27 in your settings .py, am l missing out on anything?